src/install/postgresql/find-data.mdx

```diff
@@ -0,0 +1,26 @@
+---
+componentType: default
+headingText: Find and use data
+---
+
+
+import infrastructureInstallPostgresqlDashboard from 'images/infrastructure_screenshot-full_install-postgresql-dashboard.webp'
+
+
+1. Go to **[one.newrelic.com](https://one.newrelic.com) > Add data** and type **postgresql**.
```

Comment:
```suggestion
1. Go to **[one.newrelic.com](https://one.newrelic.com) > Add data** and type **PostgreSQL**.
```

Reply:
OK, I'm going to correct it, but, actually, it doesn't matter the way you write it.